apps/dashboard/index.html

```diff
@@ -12,6 +12,11 @@
     <meta name="theme-color" content="#ffffff">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>SudoSOS</title>
+    <!-- For OpenGraph -->
+    <meta property="og:title" content="SudoSOS Dashboard">
+    <meta property="og:description" content="The bar system for study association GEWIS">
+    <meta property="og:image" content="https://github.com/GEWIS/sudosos-frontend/raw/develop/apps/dashboard/src/assets/img/bier_margin_red.png?raw=false">
```

Comment:
Would be nicer to keep the image under the domain itself instead of going to github